pls_core/src/inflections/generators/templates/indeclinable.html

```diff
@@ -5,3 +5,7 @@
   <i>'<strong>{{ word }}</strong>' is indeclinable.</i>
   {% endif -%}
 </div>
+<div class="pls-inflection-summary-definition">
```

Comment:
i think some conditional needs to be present.

otherwise we'll end up showing this in the goldendict as well since the code is shared. the other templates have that conditional iirc

Reply:
Added the same conditional logic that is present for forms with declension tables.